cmd/readiness/readiness_test.go

```diff
@@ -131,18 +131,6 @@ func TestPodReadiness(t *testing.T) {
 	})
 }
 
-func TestServerHasAlreadyStarted(t *testing.T) {
-	t.Run("Agent should report server has started", func(t *testing.T) {
-		healthDoc := readHealthinessFile("testdata/health-status-readable-state.json")
```

Comment:
Are those files still in use? If not, then they shoud be removed.

Reply:
You are right, I will